master/buildbot/process/buildstep.py

```diff
@@ -643,6 +644,9 @@ def failed(self, why):
 
     def isNewStyle(self):
         # **temporary** method until new-style steps are the only supported style
+        # Always newStyle in PY3
+        if PY3:
+            return True
```

Comment:
new-style steps and new-style classes are different things!  Unless there's some reason that old-style steps aren't supported on py3?